fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/SlicesApiRequestImpl.java

```diff
@@ -143,11 +143,6 @@ public SlicesApiRequestImpl(
             PhysicalTableDictionary tableDictionary,
             UriInfo uriInfo
     ) throws BadApiRequestException {
-        if (tableDictionary.isEmpty()) {
-            String msg = EMPTY_DICTIONARY.logFormat("Slices cannot be found. Physical Table");
-            throw new BadApiRequestException(msg);
-        }
-
```

Comment:
How do we handle this case when tableDictionary is empty?  Looks like there is one more signature of generateSlices method.

Reply:
If the dictionary is empty, we stream over the empty collection and return no slices without an error.
That's my proposed change over returning a 400 error.